src/components/deploymentConfig/DeploymentConfig.tsx

```diff
@@ -55,9 +58,11 @@ function DeploymentConfigForm({ respondOnSuccess, isUnSet }) {
     const [obj, json, yaml, error] = useJsonYaml(tempFormData, 4, 'yaml', true);
     const [chartConfigLoading, setChartConfigLoading] = useState(null)
     const [showConfirmation, toggleConfirmation] = useState(false)
+    const [showReadme, setReadme] = useState(false)
+    const [showReadmeTab,setReadmeTab] = useState('')
```

Comment:
`showReadmeTab` name convention is not correct for the string value. It feels more like boolean, rather can use just `readme` instead `showReadmeTab` and in place of `setReadmeTab` can use `setReadme`

Reply:
Done